jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/main/java/org/jbpm/console/ng/ht/client/editors/taskslist/grid/dash/DataSetTasksListGridPresenter.java

```diff
@@ -226,14 +359,14 @@ public void releaseTask( final Long taskId,
         taskOperationsService.call( new RemoteCallback<Void>() {
             @Override
             public void callback( Void nothing ) {
-                view.displayNotification( "Task Released" );
+                view.displayNotification( Constants.INSTANCE.TaskReleased() + " ("+taskId+")" );
```

Comment:
Have a look on Constants.DataSetNotFound(String dataSet), adding the taskId as a parameter would allow more flexibility when translating or changing the message if we need.